src/screens/Loading/index.js

```diff
@@ -1,31 +1,31 @@
 import * as React from 'react';
 import {ActivityIndicator, Modal, StyleSheet, View} from 'react-native';
-import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `withInteractionsManaged` has been modified. The spaces around the imported module have been removed, which is a good practice as it improves readability and consistency in the code.
```diff
- import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+ import {withInteractionsManaged} from '../../components/WithInteractionManaged';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->